freeipa/src/main/resources/application.yml

```diff
@@ -142,7 +142,9 @@ freeipa:
         - TWO_NODE_BASED
       TWO_NODE_BASED:
         - HA
+        - TWO_NODE_BASED
```

Comment:
why is this needed?

Reply:
Currently it is not needed, indeed, as you do not have means to distinguish a correctly running HA cluster from a TWO_NODE_BASED one where because of a failure in rolling upgrade has only 3 nodes. 
I will thus remove it and put back once AvailabilityType is also saved to the DB.